### PR TITLE
Leslie's changes with some nicer modularization and some proper Go syntax corrections

### PR DESCRIPTION
**Pull request by joshkergan**
Go uses capital names to indicate public funtions. And I moved the main func into the main go file, making the csv creator a proper library package.

### CODE REVIEW

**Comment by acetophore** [comment]
oh, I thought we were making the image -> csv parser a separate program

**Comment by joshkergan** [comment]
Oh, okay. Well let's talk about that then.

File: main.go [hunk 1]
```diff
@@ -1,7 +1,46 @@
 package main
 
-import "fmt"
+import (
+    "os"
+    "fmt"
+    "flag"
+    "github.com/joshkergan/TeamGandalf/imagetocsv"
```
Comment (akeizer) [inline main.go:7]: This needs to be relative doesn't it? Or if it doesn't, should we point it to the main repo?

**Review by akeizer — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (joshkergan) [reply main.go:7]: I don't know how Go resolves paths, but this is right in the case that it's setup using go get.

**Review by akeizer — APPROVED** [review]

**Pull request merged**